controllers/operatorpolicy_controller.go

```diff
@@ -216,14 +219,14 @@ func (r *OperatorPolicyReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		errs = append(errs, err)
 	}
 
+	if err := r.Status().Update(ctx, policy); err != nil {
```

Comment:
Could we avoid always updating the status if nothing changed? This could increase API server hits a lot. Maybe a reflect.DeepEqual comparing before and after would suffice.

Reply:
I can add the DeepEqual, I'm surprised controller-runtime (or whatever) wouldn't do that for us...

For extra context, the cause of this was the intervention timestamp not being in a condition. I felt like we were dealing with confusing cases where the Subscription status wasn't always updated correctly by OLM, I wanted to be more paranoid about making sure our status was accurate.